II.pass-tests/05-factorial/factorial.js

```diff
@@ -8,8 +8,13 @@
 // calculate and return the factorial of int
 // note: factorial of 0 is 1
 
-function factorial(int) {
-
+function factorial (int) {
+  if (int >= 0) {
+    var fact = 1
+    for (i = 2; i <= int; i++) {
+      fact = fact * i
```

Comment:
You can make this line more concise by changing to `fact *= i` (this just translates to 'multiply myself with i') - you can even replace this with any operator (+, -, /) and it'll do the same thing for that operation

Reply:
noted already started using it